docs_src/design/adr/system-management/0006-Metrics-Collection.md

```diff
@@ -0,0 +1,382 @@
+# EdgeX Metrics Collection
+
+## Status
+
+### Proposed
+
+!!! Note
+    This ADR initially incorporated metrics collection and control plane event processing.  The EdgeX architects felt the scope of the design was too large to cover under one ADR.  Control plane event processing will be covered under a separate ADR in the future.  For the purpose of distinction, metrics collection data and control plane events are defined below:
+
+    **Metric (or telemetry) data** is defined as the count or rate of some action, resource, or circumstance in the EdgeX instance or specific service.  Examples of metrics include:
+
+    - the number of EdgeX Events sent from core data to an application service
+    - the number of requests on a service API
+    - the average time it takes to process a message through an application service
+    - The number of errors logged by a service
+
+    **Control plane events** (CPE) are defined as `events` that occur within an EdgeX instance.  Examples of CPE include:
+
+    - a device was provisioned (added to core metadata)
+    - a service was stopped
+    - service configuration has changed
+
+    Note CPE should not be confused with core data Events.  Core data Events represent a collection (one or more) of sensor/device readings.  Core data Events represent sensing of some measured state of the physical world (temperature, vibration, etc.).  CPE represents the detection of some happening inside of the EdgeX software.
+
+!!! Info
+    Information in *italics* indicates topics for discussion without suggested proposal at this time.
+
+## Context
+
+[System Management services](../../../microservices/system-management/Ch_SystemManagement.md) (SMA and executors) currently provide a limited set of “metrics” to requesting clients (3rd party applications and systems external to EdgeX).  Namely, it provides requesting clients with service CPU and memory usage; both metrics about the resource utilization of the service itself versus metrics that are about what is happening inside of the service.  Arguably, the current system management metrics can be provided by the container engine and orchestration tools (example: by Docker engine) or by the underlying OS tooling.
+
+!!! Info
+    The SMA has been deprecated and will be removed in a future, yet named, release.

```

Comment:
This also doesn't render well. I'd suggest getting rid of the "!!! Info" and just using bold or italics to highlight.

```suggestion
    Note - The SMA has been deprecated and will be removed in a future, yet named, release.
```

Reply:
fixed